### DISCUSSION TITLE
Have Tabs render as links

### DISCUSSION THREAD

**Discussion by mattaningram** [mantinedev/mantine#4267] [Q&A]
Is there a way to make the `Tabs` component render as `a` links with an href instead of `button`? I want my users to be able to cmd/ctrl-click or right-click to open a page tab in a new browser tab, but currently they are buttons so they can't do that. Do I have to just manually remake the tab styling with custom links?

**Comment by rtivital** [comment]
Tabs component follows WAI ARIA specification, rendering tabs controls as links will be an accessibility violation.

**Reply by mattaningram** [reply]
I mean even here in GitHub the tabs are links: 
<img width="894" alt="image" src="https://github.com/mantinedev/mantine/assets/955092/010d0f9c-9b44-4040-95c7-480a685a2c47">

Unless you don't consider those tabs? I guess I think of tabs as more of a visual treatment that works with either links or buttons depending on your experience/behavioral needs.

What would you suggest using for top-level navigation that looks like tabs but should be links?

**Reply by rtivital** [reply]
I suggest building it from scratch using `nav` and `a` tags to make navigation semantic and accessible
